Source/Core/Gen4/Encounters4.hpp

```diff
@@ -20,6 +20,7 @@
 #ifndef ENCOUNTERS4_HPP
 #define ENCOUNTERS4_HPP
 
+#include <Core/Enum/Game.hpp>
```

Comment:
Don't include this header unless absolutely necessary at the header level.

Just have a `enum class Game : u32;`

Reply:
> Don't include this header unless absolutely necessary at the header level.
> 
> Just have a `enum class Game : u32;`

done